### PR TITLE
Try-it-yourself

### PR DESCRIPTION
**Pull request by shaurmanchic**

### CODE REVIEW

**Comment by ruslanmalogulko** [comment]
@shaurmanchic conflicts detected

File: try_it_results.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (dmarynych) [inline try_it_results.json:1]: should this file be generated on ci side?

Comment (ruslanmalogulko) [reply try_it_results.json:1]: I guess it should be in gitignore

Comment (shaurmanchic) [reply try_it_results.json:1]: Yep, I included it as a reference

**Review by ruslanmalogulko — APPROVED** [review]
🔥🔥🔥

**Pull request merged**